core/domain/stats_domain.py

```diff
@@ -635,17 +753,19 @@ def __eq__(self, other):
             )
         return NotImplemented # https://stackoverflow.com/a/44575926
 
-    def __hash__(self):
+    def __hash__(self) -> int:
         """Disallow hashing SessionStateStats since it is mutable by design."""
         raise TypeError('%s is unhashable' % self.__class__.__name__)
 
     @classmethod
-    def create_default(cls):
+    def create_default(cls) -> SessionStateStats:
         """Creates a SessionStateStats domain object with all values at 0."""
         return cls(0, 0, 0, 0, 0, 0)
 
     @classmethod
-    def from_dict(cls, session_state_stats_dict):
+    def from_dict(
+        cls, session_state_stats_dict:
```

Comment:
Is it possible to have session_state_stats_dict on the next line, or would that make it too long? Bit weird to separate the arg and its type.

Reply:
Done